pkgs/applications/misc/qolibri/default.nix

```diff
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
```

Comment:
```suggestion
mkDerivation rec {
```

And the application will automatically be wrapped (currently fails at runtime).
Preferably in a separate commit.

Reply:
Thanks. I didn't see a failure at runtime for some reason, but I have updated it to use `mkDerivation`.